client/human/chmain.h

```diff
@@ -1,8 +1,7 @@
 #ifndef _CHMAIN_H
 #define _CHMAIN_H
 
-#include <string>
-#include <vector>
+#include "../../util/ConfigFileProcess.h"
```

Comment:
>  And a question for Marcel: it seems that this chmain.h header is not actually needed (and currently does not seem to be used at all in master)

It is used in `client/human/chmain.mm`.  However I'm not sure how this file interacts with the rest of the chmain files.  But I'm quite sure that OSX doesn't use the main method in chmain.cpp at all.